doc/tutorials/prox_nn_group_lasso.rst

```diff
@@ -191,8 +180,21 @@ Minimizing over :math:`n` then over :math:`u`, thanks to [`1 <https://math.stack
 
 Where :math:`v^+` is :math:`v` restricted to its positive coordinates.
 
+Case :math:`|| w  || \ne 0`
+-----------------------------------
+The subdifferential in that case is :math:`\lambda w / {|| w ||} + C_1 \times \ldots \times C_g` where :math:`C_i = {0}` if :math:`w_i > 0` and :math:`C_i = mathbb{R}_-` otherwise (:math:`w_i =0`).
```

Comment:
```suggestion
The subdifferential in that case is :math:`\lambda w / {|| w ||} + C_1 \times \ldots \times C_g` where :math:`C_j = {0}` if :math:`w_j > 0` and :math:`C_j = mathbb{R}_-` otherwise (:math:`w_j =0`).
```